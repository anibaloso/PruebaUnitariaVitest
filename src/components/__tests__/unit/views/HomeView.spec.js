
import HomeView from '@/views/HomeView.vue'
import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'


describe('HomeView.vue', () => {
    it('El HTML se mantiene en las pruebas', () => {
        const wrapper = mount(HomeView)
        expect(wrapper.html()).toMatchSnapshot()
    })
})