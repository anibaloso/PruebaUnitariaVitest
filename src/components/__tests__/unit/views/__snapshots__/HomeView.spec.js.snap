// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView.vue > El HTML se mantiene en las pruebas 1`] = `
"<main>
  <h1>Vista Home</h1>
</main>"
`;
